moveit_simple/src/moveit_simple.cpp

```diff
@@ -895,7 +909,30 @@ bool Robot::toJointTrajectory(const std::string traj_name,
   return true;
 }
 
+bool Robot::computeIKTransforms()
```

Comment:
This method wraps an existing method called `knowsFrameTransform`...is there a reason you just didn't re-use the name?

Reply:
It does more.